CONTRIBUTING-de.md

```diff
@@ -72,43 +72,43 @@ Beispiel:
 - Keine Leerzeichen zwischen `]` und `(` einfügen:
 
 ```
-FALSCH : * [Noch ein tolles Buch] (http://example.com/book.html)
+FALSCH: * [Noch ein tolles Buch] (http://example.com/book.html)
```

Comment:
Not sure about these re-formattings. They were like this in the original english file too, presumably to have the examples aligned so that you can spot the differences more easily. I would keep the aligned `:`s.

Reply:
Don't want to be too small minded here, but [this article](https://en.wikipedia.org/wiki/Colon_(punctuation)#:~:text=In%20modern%20English%2Dlanguage%20printing,space%20is%20placed%20after%20it.&text=One%20or%20two%20spaces%20may,been%20used%20after%20a%20colon.) and a little more research would let me think a space before a colon is incorrect.

Reply:
What do you think about adding an extra space AFTER the colon instead?

Reply:
I wouldn't do that, since it's an older convention with the following reason:

> Back in the typewriter day, when fading ink ribbons could result in commas being mistaken for periods and vice versa, typists were taught to insert 2 spaces after the period to differentiate between the two. The same happened with colons and semicolons: 2 spaces were left after a colon; 1 space after a semicolon.

Reply:
@davorpa what's your take on that? In the original CONTRIBUTING.md file, it is `BAD :` and `GOOD:`. I am pretty sure this is to make sure the examples align, and IMO it totally makes sense that they align.

Reply:
You are right. Since is a preformatted block... to have same length of characters and align the important part of example.

Reply:
Alright :) There you go :)